.rubocop.yml

```diff
@@ -1,6 +1,9 @@
 # TODO: remove this
 inherit_from: .rubocop_todo.yml
 
+AllCops:
+  TargetRubyVersion: 2.3
```

Comment:
This was set to the default 2.0 which won't let some of the code in here through. I set this to the lowest tested Ruby version.